interactive/interactive/tools.py

```diff
@@ -46,8 +46,13 @@ def open_json(filename):
     with open(path) as f:
         dicti = json.loads(f.read())
     return dicti
-
-
+def direct_open_json(filePath):
+    with open(filePath) as f:
+        dicti = json.loads(f.read())
+    return dicti
+def direct_dump_json(filePath, content):
+    with open(filePath, 'w') as f:
+        json.dump(content, f, indent=4)
```

Comment:
let's remove the indent. We can always pretty print if we need to when debugging an issue, but let's not force the files to be larger than they need to be.